bgpd/bgpd.c

```diff
@@ -3542,6 +3542,9 @@ static struct bgp *bgp_create(as_t *as, const char *name,
 	if (cmd_domainname_get())
 		bgp->peer_self->domainname =
 			XSTRDUP(MTYPE_BGP_PEER_HOST, cmd_domainname_get());
+	SET_FLAG(bgp->peer_self->cap, PEER_CAP_AS4_RCV);
+	SET_FLAG(bgp->peer_self->cap, PEER_CAP_AS4_ADV);
```

Comment:
Looks correct, but could you add a comment above saying that this is for BMP Loc-RIB? It's very non-obvious otherwise.